docker.go

```diff
@@ -561,7 +561,7 @@ func (c *DockerContainer) CopyFileFromContainer(ctx context.Context, filePath st
 
 // CopyDirToContainer copies the contents of a directory to a parent path in the container. This parent path must exist in the container first
 // as we cannot create it
-func (c *DockerContainer) CopyDirToContainer(ctx context.Context, hostDirPath string, containerParentPath string, fileMode int64) error {
+func (c *DockerContainer) CopyDirToContainer(ctx context.Context, hostDirPath, containerParentPath string, fileMode int64) error {
```

Comment:
Let's not add this to the PR, as it's not related to the change. Also, I have a preference on being explicit about param types

Reply:
These changes were to satisfy `golangci-lint` which failed in ci requiring a run of `gofumpt` which I ran with `-extras` that brought in the parameter simplification.

When running the `golangci-lint` locally to check I got additional failures requiring `gci` run.

So these are just to satisfy the linter, given that context are we ok to keep?

Reply:
This is my output of running `make lint`:

```
golangci-lint run --out-format=github-actions --path-prefix=. --verbose -c /Users/mdelapenya/sourcecode/src/github.com/testcontainers/testcontainers-go/.golangci.yml --fix
INFO [config_reader] Used config file .golangci.yml 
INFO [lintersdb] Active 13 linters: [errcheck errorlint gci gocritic gofumpt gosimple govet ineffassign misspell nonamedreturns staticcheck testifylint unused] 
INFO [loader] Go packages loading at mode 575 (deps|exports_file|files|imports|name|compiled_files|types_sizes) took 3.022026917s 
INFO [runner/filename_unadjuster] Pre-built 0 adjustments in 10.201292ms 
INFO [linters_context/goanalysis] analyzers took 10.898360252s with top 10 stages: buildir: 6.216008661s, fact_deprecated: 623.659291ms, printf: 538.783786ms, ctrlflow: 536.868249ms, fact_purity: 493.425624ms, inspect: 416.121845ms, nilness: 359.638251ms, typedness: 260.646161ms, SA5012: 257.968917ms, gocritic: 155.162958ms 
INFO [runner] fixer took 0s with no stages        
INFO [runner] Issues before processing: 112, after processing: 0 
INFO [runner] Processors filtering stat (out/in): cgo: 112/112, filename_unadjuster: 112/112, identifier_marker: 112/112, skip_files: 112/112, skip_dirs: 112/112, autogenerated_exclude: 112/112, exclude-rules: 20/112, nolint: 0/20, path_prettifier: 112/112, exclude: 112/112 
INFO [runner] processing took 6.272461ms with stages: nolint: 3.900209ms, identifier_marker: 891.542µs, exclude-rules: 753.667µs, path_prettifier: 364.583µs, autogenerated_exclude: 319.292µs, skip_dirs: 26.876µs, fixer: 8µs, cgo: 3.541µs, filename_unadjuster: 1.958µs, max_same_issues: 541ns, severity-rules: 375ns, exclude: 334ns, uniq_by_line: 292ns, source_code: 250ns, diff: 209ns, max_from_linter: 208ns, sort_results: 167ns, skip_files: 167ns, path_shortener: 84ns, path_prefixer: 83ns, max_per_file_from_linter: 83ns 
INFO [runner] linters took 2.304556292s with stages: goanalysis_metalinter: 2.29824575s 
INFO File cache stats: 76 entries of total size 493.2KiB 
INFO Memory: 55 samples, avg is 165.3MB, max is 502.4MB 
INFO Execution took 5.346137458s
```

It does not raise errors.

Given that, I'd prefer keeping the type to all the params, which is consistent everywhere else. 🙏